opennms-services/src/main/java/org/opennms/netmgt/poller/QueryManagerDaoImpl.java

```diff
@@ -362,4 +375,36 @@ public void updateLastGoodOrFail(PollableService pollableService, PollStatus sta
         }
     }
 
+    @Override
+    public void persistDeviceConfig(PollableService service, Map<String, Object> attributes, byte[] deviceConfigBytes) {
```

Comment:
Please add a test, that checks the persisting logic (i.e. the version incrementation logic.)